doc/source/starting.rst

```diff
@@ -46,6 +46,31 @@ System Wide (for the current user)
 #. Install it from `PyPI <https://pypi.org/project/montepy>`_ by running ``pip install montepy``. 
    You may need to run ``pip install --user montepy`` if you are not allowed to install the package.
 
+Installing Optional Dependencies
```

Comment:
The addition of this section is really helpful! Thank you also for providing short descriptors of these options as well.